src/app/components/fees-row/fees-row.tsx

```diff
@@ -46,7 +49,12 @@ export function FeesRow(props: FeeRowProps): React.JSX.Element {
   }, [convertCryptoCurrencyToUsd, feeCurrencySymbol, feeField.value]);
 
   useEffect(() => {
-    if (hasFeeEstimates && !feeField.value && !isCustom) {
+    if (props.defaultFeeValue) {
+      feeHelper.setValue(
+        convertAmountToBaseUnit(new BigNumber(Number(props.defaultFeeValue)), STX_DECIMALS)
+      );
+      feeTypeHelper.setValue(FeeTypes[FeeTypes.Custom]);
+    } else if (hasFeeEstimates && !feeField.value && !isCustom) {
```

Comment:
I think this would read better as it's own if statement rather than useing `else if`.